src/frontend/js/pages/TeacherDashboardContractsLayout/TeacherDashboardContracts/index.tsx

```diff
@@ -41,7 +41,7 @@ const TeacherDashboardContracts = () => {
     defaultPage: page ? parseInt(page, 10) : 1,
     pageSize: PER_PAGE.teacherContractList,
   });
-  const { organizationId } = useParams<TeacherDashboardContractsParams>();
+  const { organizationId: routeOrganizationId } = useParams<TeacherDashboardContractsParams>();
```

Comment:
Not quit sure to understand why you need to remove this.

Reply:
I didn't remove anything.
I just rename `organizationId` into `routeOrganizationId`. I did it because i get lost between organizationId comming from route, query and filter in list.

It's obvious l.91, when we see `hideFilterOrganization={!!routeOrganizationId}` that the filter is hidden when the route is outside of organization context.